tasks/task179_participant_extraction.json

```diff
@@ -0,0 +1,2577 @@
+            "output": [
+                "children,with,malaria"
+            ]
+        },
+        {
+            "input": "Both groups received 24 months of 1,200 mg of calcium and 400 IU of vitamin D daily and 35 mg of risedronate weekly , and the exercise group additionally had ST exercises twice weekly .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "Daily supplementation significantly increased plasma concentrations of alpha-tocopherol and beta-carotene in the supplemented group ( Delta 14.4 ( SE 3.2 ) and 0.4 ( se 0.1 ) micromol/l ; P < 0.001 and P < 0.01 ) .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "We performed a prospective randomized trial to evaluate the optimum duration of stenting after endopyelotomy .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "A new immunomodulating agent , bestatin ( INN : Ubenimex has low toxicity even after long-term oral administration and has significant modifications in immunological response .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "We expect the patients in the active recommendations group to experience a greater improvement in their depressive symptoms .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "Patients were scanned twice , under either an acute dose of Fluoxetine or placebo in a double-blind , placebo-controlled randomized design .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "We discuss the theoretical relevance of these findings and educational implications .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "The patients were followed for change in outcome measures during the five-month study period .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "Toxicities , including thromboembolism , and overall survival were similar .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "Non-inferiority was defined as an upper 95 % confidence limit ( CL ) for the difference in success of eradication in the amoxicillin and penicillin V treatment groups of < or=10 % .",
+            "output": [
+                "not found"
+            ]
+        },
+        {
+            "input": "In conclusion , in 60-year-old subjects antioxidant supplementation had no effect on the exercise-induced increase in oxidative stress as measured by free radical products of antipyrine .",
+            "output": [
+                "60-year-old,subjects"
```

Comment:
why is there a comma between "subjects" and "60-year-old"?

Reply:
Umm, I don't know how to present the outputs. The annotation they have is a binary array with a 1 for each token that matches and 0 for others. So I thought of presenting them as separate tokens. I'd be glad if you have any suggestions.

Reply:
I see these options: 
 (1) Expect only one of the responses: Basically, a system that returns one of the correct spans gets the full credit. Requires minor changes in the instructions and also having list of answers, like this: 
```
"output": [
                "60-year-old",
                 "subjects"
]
```
 (2) Include all the valid combinations. For example:  
```
 "output": [
                "60-year-old,subjects", 
                "subjects,60-year-old"
]
```
 (3) define an expected order in the definition (e.g,. in the same order which they appear in the text).

What do you think?

Reply:
I think the third one makes more sense.